benchmarks/ner_embeddings/project.yml

```diff
@@ -0,0 +1,278 @@
+        --nlp.batch_size ${vars.batch_size}
+        --system.seed ${vars.seed}
+        --gpu-id ${vars.gpu_id}
+        --code scripts/custom_components.py
+    deps:
+      - corpus/${vars.dataset}-train.spacy
+      - corpus/${vars.dataset}-dev.spacy
+    outputs:
+      - training/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/model-last
+      - training/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/model-best
+
+  - name: "train-hash"
+    help: "Train NER model with different number of hash functions."
+    script:
+      - >-
+        python -m spacy train
+        configs/${vars.ner_config}.cfg
+        --nlp.lang ${vars.language}
+        --output training/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/
+        --paths.train corpus/${vars.dataset}-train.spacy
+        --paths.dev corpus/${vars.dataset}-dev.spacy
+        --paths.vectors ${vars.vectors}
+        --paths.tables ${vars.tables_path}/${vars.dataset}/${vars.dataset}-train.tables
+        --components.tok2vec.model.embed.include_static_vectors ${vars.include_static_vectors}
+        --components.tok2vec.model.embed.num_hashes ${vars.num_hashes}
+        --nlp.batch_size ${vars.batch_size}
+        --system.seed ${vars.seed}
+        --gpu-id ${vars.gpu_id}
+        --code scripts/custom_components.py
+    deps:
+      - corpus/${vars.dataset}-train.spacy
+      - corpus/${vars.dataset}-dev.spacy
+    outputs:
+      - training/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/model-last
+      - training/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/model-best
+
+  - name: "evaluate"
+    help: "Evaluate NER model."
+    script:
+      - mkdir -p ${vars.metrics_dir}/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/
+      - >-
+        python -m spacy evaluate
+        training/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/model-best
+        corpus/${vars.dataset}-test.spacy
+        --output ${vars.metrics_dir}/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/scores-test.json
+        --gpu-id ${vars.gpu_id}
+        --code scripts/custom_components.py
+      - >-
+        python -m spacy evaluate
+        training/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/model-best
+        corpus/${vars.dataset}-dev.spacy
+        --output ${vars.metrics_dir}/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/scores-dev.json
+        --gpu-id ${vars.gpu_id}
+        --code scripts/custom_components.py
+    deps:
+      - training/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/model-best
+      - corpus/${vars.dataset}-dev.spacy
+    outputs:
+      - ${vars.metrics_dir}/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/scores-test.json
+      - ${vars.metrics_dir}/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/scores-dev.json
+
+  - name: "evaluate-seen-unseen"
```

Comment:
The output of this is not that clear - maybe add something like dividers to the output to clearly show what the numbers relate to in terms of dev/test seen/unseen? Right now you basically have to read the Python command to understand what the numbers refer to.

Reply:
I just split it into two commands `evaluate-seen-only` and `evaluate-unseen-only`. I think this makes it more clear.